insights/combiners/cpu_vulns_all.py

```diff
@@ -24,7 +14,29 @@
 @combiner(CpuVulns)
 class CpuVulnsAll(dict):
     """
-    This combiner provides an interface to CPU vulnerabilities parsers.
+    Class to capsulate the parsers of cpu_vulns, which includes all files in
+    ``/sys/devices/system/cpu/vulnerabilities/*``. Files information will be
+    stored in a list of dictionaries, each dictionary is for one file, the
+    dictionary key is the file name, dictionary value is the file content.
+
+    Sample output for files:
+        ``/sys/devices/system/cpu/vulnerabilities/spectre_v1``::
+            Mitigation: Load fences
+
+         ``/sys/devices/system/cpu/vulnerabilities/meltdown``::
```

Comment:
One more space is added. please remove.
And please remove one ":", keep one here is ok.

Reply:
ok, I fixed it as you suggested.